commercial/app/views/contentapi/items.scala.html

```diff
@@ -28,7 +28,8 @@ <h3 class="commercial__title">
                     </a>
                 </div>
                 <div class="commercial__body">
-                    <ul class="u-unstyled l-row l-row--cols-4 l-row--layout-m fc-slice">
+                    <div class="fc-slice-wrapper">
```

Comment:
No. Although, it seems there are some templates we want to change that don't use fc-slice in the first place. So I may need another solution anyway.